create-liquid-app

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+# ensure that .log directory exists
+readonly log_dir="./.log"
+mkdir -p $log_dir
+
+readonly nigiri_config_path=$(realpath ./.nigiri)
+
+# start elementsd, esplora (et al)
+nigiri --datadir $nigiri_config_path start --liquid
+
+# bring in environment variables generated by nigiri
+export $(cat .nigiri/.env | xargs)
+
+readonly mint_amount=1000000
+mint_address=$(nigiri --datadir $nigiri_config_path rpc --liquid getnewaddress)
+sleep 2
+mint_response=$(nigiri --datadir $nigiri_config_path mint $mint_address $mint_amount "Tether USD" "USDt")
+mint_asset_id=$(echo $mint_response | sed -n -e 's/^.*\(asset: \)\(.*\)\( issuance_txin: .*$\)/\2/p')
+
+echo "Minted $mint_amount USDt with asset ID $mint_asset_id to elementsd wallet address $mint_address"
+
+# start bobtimus
+readonly elementsd_rpc_user="admin1"
+readonly elementsd_rpc_password="123"
+RUST_LOG=info cargo run --bin fake_bobtimus -- \
+        --elementsd http://$elementsd_rpc_user:$elementsd_rpc_password@127.0.0.1:$LIQUID_NODE_PORT \
+        --usdt $mint_asset_id > $log_dir/bobtimus 2>&1 &
+bobtimus_pid=$!
+
+echo "Starting bobtimus with pid $bobtimus_pid"
+
+# start frontend
+readonly chopsticks_url="http://127.0.0.1:$LIQUID_CHOPSTICKS_PORT"
+cd waves && ESPLORA_URL=$chopsticks_url REACT_APP_ESPLORA_URL=$chopsticks_url yarn start
```

Comment:
I am not sure how often we will need to start / stop things but if that turns out to be the case (with your emacs setup for example) then it might be better if this script where to export _functions_ to the current shell instead of running the actual commands.

I am thinking of something like:

- `start_bobtimus`
- `stop_bobtimus`
- `start_frontend`
- etc

That way, running the script will just the define the functions correctly and it is quite easy to start & stop things :)

Reply:
> with your emacs setup for example

I managed to fix that!

I like your idea, although I think there's still value in having one command that starts everything up, since the order in which things are started matters. That one could be a composition of the functions that you suggest, which we can still expose separately.